### PR TITLE
Jit64: Fallback support of branching instructions

### PR DESCRIPTION
**Pull request by degasus**

### CODE REVIEW

File: Source/Core/Core/PowerPC/Jit64/Jit.cpp [hunk 1]
```diff
@@ -245,6 +245,26 @@ void Jit64::WriteCallInterpreter(UGeckoInstruction inst)
...
+		{
+			MOV(32, R(RSCRATCH), PPCSTATE(npc));
+			MOV(32, PPCSTATE(pc), R(RSCRATCH));
+			WriteExceptionExit();
+		}
+		else
+		{
+			MOV(32, R(RSCRATCH), PPCSTATE(npc));
+			SUB(32, R(RSCRATCH), Imm32(js.compilerPC + 4));
+			TEST(32, R(RSCRATCH), R(RSCRATCH));
```
Comment (Tilka) [inline Source/Core/Core/PowerPC/Jit64/Jit.cpp]: You can just drop the TEST.

File: Source/Core/Core/PowerPC/Jit64/Jit.cpp [hunk 2]
```diff
@@ -236,7 +236,7 @@ void Jit64::WriteCallInterpreter(UGeckoInstruction inst)
 {
 	gpr.Flush();
 	fpr.Flush();
-	if (js.isLastInstruction)
+	if (js.op->opinfo->flags & FL_ENDBLOCK)
```
Comment (phire) [inline Source/Core/Core/PowerPC/Jit64/Jit.cpp]: Does this work with single stepping, when every instruction is at the end of the block?

Comment (degasus) [inline Source/Core/Core/PowerPC/Jit64/Jit.cpp]: I think so, we  should hit https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/Jit64/Jit.cpp#L859
Else all non-fallbacked instructions would also break.

**Comment by degasus** [comment]
Any remaining comments?

**Pull request merged**